docs/book/src/topics/capibmadm/vpc/key.md

```diff
@@ -0,0 +1,53 @@
+## VPC key Commands
+
+ ### 1. capibmadm vpc key create
+
+ #### Usage: 
+ Create a key in the VPC environment.
+
+ #### Environmental Variable:
+ IBMCLOUD_API_KEY: IBM Cloud api key.
+
+ #### Arguments:
+
+  --name: The name of the key. 
+
+  --resource-group-name: Optional VPC resource group name.
```

Comment:
```suggestion
  --resource-group-name: VPC resource group name.
```

Reply:
@Madhan-SWE if `--resource-group-name` is optional then lets drop it from example commands as well.